pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -243,7 +257,12 @@ func shutdownAppsGlobal(ctx *zedagentContext) {
 var baseOSPrevConfigHash []byte
 
 func parseBaseOS(getconfigCtx *getconfigContext,
-	config *zconfig.EdgeDevConfig) {
+	config *zconfig.EdgeDevConfig) (activateNewBaseOSFlag bool) {
+	// activateNewBaseOSFlag is set to true if we need to activate a new baseOS:
+	// 1. If the config has a new baseOS image with the activate flag set to true
+	// 2. If the config has a previous baseOS image, but the activate flag is _switched_ from false to true
+	// We don't care if the active flag already was true, as that means that the baseOS is already active
```

Comment:
Instead of "is active" I would say that the process of activating it has already started.

Reply:
Ok, I will fix it now.

Reply:
done